aggregator/src/aggregator.rs

```diff
@@ -2830,7 +2838,7 @@ impl VdafOps {
         Ok(())
     }
 
-    /// Implements the `/aggregate_share` endpoint for the helper, described in §4.4.4.3
+    /// Implements the `tasks/{task-id}/aggregate_shares` endpoint for the helper, described in §4.4.4.3
```

Comment:
The section reference is out of date (I would think), let's just delete it
```suggestion
    /// Implements the `tasks/{task-id}/aggregate_shares` endpoint for the helper
```